server/index.js

```diff
@@ -2,13 +2,14 @@ const express = require('express')
 const app = express()
```

Comment:
Are you using no-semicolon style or semicolon style? I think it will help to be consistent or set up a linter.